CHARTER.md

```diff
@@ -0,0 +1,113 @@
+# Technical Charter (the "Charter") for Besu a Series of LF Projects, LLC
+
+**Amended June \_\_, 2025**
+
+This Charter sets forth the responsibilities and procedures for technical contribution to, and oversight of, the Besu open source project, which has been established as Besu a Series of LF Projects, LLC (the "Project").  LF Projects, LLC ("LF Projects") is a Delaware series limited liability company. All contributors (including committers, maintainers, and other technical positions) and other participants in the Project (collectively, "Collaborators") must comply with the terms of this Charter.
+
+1. **Mission and Scope of the Project**
+
+      1. The mission of the Project is to develop an Ethereum execution client.
+
+      2. The scope of the Project includes collaborative development under the Project License (as defined herein) supporting the mission, including documentation, testing, integration and the creation of other artifacts that aid the development, deployment, operation or adoption of the open source project.
+
+2. **Maintainer Oversight**
+
+      1. The maintainers of the project (collectively, the "Maintainers")  will be responsible for all technical oversight of the open source Project.
+
+      2. At the inception of the project and going forward, the Maintainers of the Project will be as set forth within the "MAINTAINERS" file within the Project's code repository. The Maintainers may choose an alternative approach for determining the voting members of the Project (such as the establishment of a technical steering committee for project technical oversight), and any such alternative approach will be documented in the MAINTAINERS file.  Any meetings of the Maintainers are intended to be open to the public, and can be conducted electronically, via teleconference, or in person.
+
+      3. Sub-projects generally will involve Contributors and Maintainers. The Maintainers  may adopt or modify roles so long as the roles are documented in the MAINTAINERS file. Unless otherwise documented:
+
+         1. Contributors include anyone in the technical community that contributes code, documentation, or other technical artifacts to the Project;
+
+         2. Maintainers are Contributors who have earned the ability to modify ("commit") source code, documentation or other technical artifacts in a project's repository; and  
```

Comment:
documentation repo has separate maintainers so maybe that needs to be discussed separately